House_Builder.java

```diff
@@ -0,0 +1,17 @@
+
+public interface House_Builder {
+
+	public void buildarea();

```

Comment:
All methods define in an interface are public. It is redundant to use the keyword `public`.